задание 5.2 вариант 9/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace задание_5._2_вариант_9
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Получить из слова кратер слова катер и река");
+
+            string str = "";
+
+            str = "Кратер";//к-0 р-1 а-2 т-3 е-4 р-5
```

Comment:
В задании дано слово с маленькой буквы
```suggestion
            str = "кратер";//к-0 р-1 а-2 т-3 е-4 р-5
```